dashboard/app/models/user.rb

```diff
@@ -507,21 +506,15 @@ def sanitize_race_data_set_urm
   # Implement validation that refuses to set admin:true attribute unless
   # there's a Code.org google sso option present.  Unmigrated users are
   # not allowed to be admins.
+  validate :enforce_google_sso_for_admin
   def enforce_google_sso_for_admin
-    unless migrated?
-      self.admin = false
-      return
-    end
+    if admin
```

Comment:
super nit: we can return early rather than wrapping this entire method in a conditional

```
def enforce_google_sso_for_admin
  return unless admin

  # other validation code
end
```